pkg/athena/datasource.go

```diff
@@ -186,10 +185,20 @@ func parseArgs(queryArgs json.RawMessage) (sqlds.Options, error) {
 		}
 	}
 	options := sqlds.Options{}
-	options[models.Region] = args.Region
-	options[models.Catalog] = args.Catalog
-	options[models.Database] = args.Database
-	options[models.ResultReuseEnabled] = fmt.Sprintf("%t", args.ResultReuseEnabled)
-	options[models.ResultReuseMaxAgeInMinutes] = fmt.Sprintf("%d", args.ResultReuseMaxAgeInMinutes)
+	if args.Region != "" {
```

Comment:
this kind of thing failed on an e2e test? we can check this in a unit test. (we can do this in a separate commit)

Reply:
*separate PR